apps/api/src/app/environments-v1/usecases/output-renderers/expand-email-editor-schema.usecase.ts

```diff
@@ -157,27 +214,7 @@ export class ExpandEmailEditorSchemaUsecase {
   }
 
   private isAVariableNode(newNode: TipTapNode): newNode is TipTapNode & { attrs: { id: string } } {
-    return newNode.type === 'payloadValue' && newNode.attrs?.id !== undefined;
```

Comment:
No idea what is `payloadValue` but I was not able to find it anywhere in the `TipTapNode`